scalableminds/webknossos
# Poll wk version to notify during upgrade

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- open help menu in navbar
- check in dev tools that every 2s a request is made to build info
- close the menu
- --> requests should stop
- update the server (or monkeypatch it somehow) --> a message should appear in the help menu (I tested this and it worked)


------
